packages/devtools_app/lib/src/code_size/code_size_table.dart

```diff
@@ -163,6 +162,7 @@ class CodeSizeDiffTable extends StatelessWidget {
   final ColumnData<TreemapNode> sortColumn;
   final List<ColumnData<TreemapNode>> columns;
 
+  // TODO(petedjlee): Ensure the sort column is descending with the absolute value of the size.
```

Comment:
I'm not sure if we want to do this for combo mode, probably only for increase only and decrease only

Reply:
I agree. Specifying it for decrease only